src/app/home/home.component.html

```diff
@@ -16,103 +16,204 @@
-      </div>
-      <div class="col-md-3 hidden-xs">
-        <img id="truck" src="../assets/images/dockstore/truck.png">
+
+      <div fxLayout="column wrap" fxLayoutAlign="center center" fxLayoutGap="20px">
+        <h3 class="search-title">New to Dockstore? Start here</h3>
+        <div fxLayout="row wrap" fxLayoutAlign="center center" fxLayoutGap="20px" class="pb-5">
+          <button mat-raised-button>
+            <a href="#developerIntroduction" class="button-link">Developers</a>
+          </button>
+          <button mat-raised-button>
+            <a href="#scientistIntroduction" class="button-link">Scientists</a>
+          </button>
+        </div>
+
+        <!-- <div fxLayout="row wrap" fxLayoutAlign="center" fxLayoutGap="20px">
+          <img fxFlexAlign="end" src="../assets/images/dockstore/ship.png">
+          <img fxFlexAlign="end" src="../assets/images/dockstore/truck.png">
+        </div> -->
+
       </div>
     </div>
   </div>
 </div>
 
 <div class="header-lower">
-  <div class="container">
-    <img class="blue-rectangle" src="../assets/images/dockstore/rectangle.png">
-    <div class="row header-lower">
-      <div class="col-md-6 col-sm-12">
-        <h4 class="title caption">
-          Dockstore, developed by the <a class="white-link" href="https://www.cancercollaboratory.org/" target="_blank">Cancer Genome
-          Collaboratory</a>, is an open platform used by the <a class="white-link" href="https://genomicsandhealth.org/" target="_blank">GA4GH</a>
-          for sharing Docker-based tools described with the <a class="white-link" href="http://www.commonwl.org/" target="_blank">Common
-          Workflow Language (CWL)</a>, the <a class="white-link" href="https://software.broadinstitute.org/wdl/" target="_blank">Workflow
-          Description Language (WDL)</a>, or <a class="white-link" href="https://www.nextflow.io/" target="_blank">Nextflow (NFL)</a>
-        </h4>
-      </div>
-      <div class="col-md-3 col-sm-12 row-center">
-        <a class='btn youtube' role="button" (click)="openYoutube()">
+  <img class="blue-rectangle" src="../assets/images/dockstore/rectangle.png">
+
+  <div fxFlex fxLayout="column" fxLayoutAlign="center center">
+      <a class='btn youtube' role="button" (click)="openYoutube()">
           <div>
             <mat-icon>play_circle_outline</mat-icon>
             <span>&nbsp;Video Overview</span>
           </div>
         </a>
+  </div>
+</div>
+
+<div id="developerIntroduction" class="container">
+  <h2 class="text-center">Developers</h2>
+  <p class="lead text-center">
+    Make your tools more accessible to the scientific community.
+  </p>
+  <div fxLayout="row wrap" fxLayoutGap="30px" fxLayoutAlign="center center" class="developer-holder flex-column-on-mobile">
+    <div fxFlex>
+      <div fxLayout="column">
+        <mat-icon fxFlexAlign="center" style="font-size: 50px; height: 50px">account_balance</mat-icon>
```

Comment:
style repeated enough to deserve a class (which applies the style to all mat-icons within the div)